Sources/TuistKit/Lint/CodeLinter.swift

```diff
@@ -42,15 +45,26 @@ class CodeLinter: CodeLinting {
 
     private func swiftLintConfigPath(path: AbsolutePath) -> AbsolutePath? {
         guard let rootPath = rootDirectoryLocator.locate(from: path) else { return nil }
-
         return ["yml", "yaml"].compactMap { (fileExtension) -> AbsolutePath? in
-            let swiftlintPath = rootPath.appending(RelativePath("\(Constants.tuistDirectoryName)/swiftlint.\(fileExtension)"))
+            let swiftlintPath = rootPath.appending(RelativePath("\(Constants.tuistDirectoryName)/.swiftlint.\(fileExtension)"))
```

Comment:
I took me a few seconds to realize that the dot was not there. I think the standard is with a `.`
What do you think @laxmorek ?

Reply:
Right. `dot` should be here :)